src/LinkedList.ts

```diff
@@ -10,34 +10,22 @@ export class LinkedList {
     private nodes: Node[];
     private canvas: fabric.Canvas;
 
-    constructor(canvasEl: HTMLCanvasElement, values: number[] = []) {
+    constructor(canvasEl: HTMLCanvasElement) {
 
         this.canvas = new fabric.Canvas(canvasEl);
         this.canvas.selection = false;
 
         // this allows us to draw circles using the coordinates of their centers
         fabric.Object.prototype.originX = fabric.Object.prototype.originY = 'center';
 
-        this.nodes = [];
-        for (let num of values) {
-            this.nodes.push(new Node(num, this.canvas));
-        }
-
-        for (let i = 0; i < this.nodes.length - 1; ++i) {
-            this.nodes[i].next.set(this.nodes[i + 1]);
-        }
-
-        this.globalVars = {
-            head: new Variable("head", this.canvas),
-        }
-        this.globalVars.head.set(this.nodes[0]);
```

Comment:
lets leave this in here--I want to add a feature where the user can create a list just by typing in a list of numbers like `1, 5, 3, 7` instead of needing to construct everything by hand. And even if we don't do that, we still want to be able to start with a list for certain activities we will ask students to do (e.g. remove a node from an existing list we give them).